perses/annihilation/relative.py

```diff
@@ -104,6 +106,10 @@ def __init__(self,
         neglected_old_angle_terms : list
             list of indices from the HarmonicAngleForce of the old_system for which the geometry engine neglected.
             Hence, these angles must be alchemically deleted for the unique old atoms (reverse lambda protocol)
+        special_new_terms : dict, default None
+            dict of new valence/1,4 terms used for stereochemical/ring strucure stability
```

Comment:
`strucure` -> `structure`

Can we also add more documentation about how these terms are used? As you note above:
> special terms is a dict of omitted bonds, angles, and torsions (which are annealed in through AIS), as well as extra angles and torsions that bias RJ proposals (which are annealed off during the AIS protocol);

Reply:
will do!